proof/invariant-abstract/AARCH64/ArchSchedule_AI.thy

```diff
@@ -37,17 +37,11 @@ lemma arch_stt_invs [wp,Schedule_AI_asms]:
 
 lemma arch_stt_tcb [wp,Schedule_AI_asms]:
   "\<lbrace>tcb_at t'\<rbrace> arch_switch_to_thread t' \<lbrace>\<lambda>_. tcb_at t'\<rbrace>"
```

Comment:
nitpick: use invariant notation for the lemmas in this file?

Reply:
can do